exporter.go

```diff
@@ -209,13 +218,14 @@ func (b *Exporter) Listen(e <-chan Events) {
 					}
 				}
 			} else {
+				eventsUnmapped.Inc()
 				metricName = escapeMetricName(event.MetricName())
 			}
 
 			switch event.(type) {
 			case *CounterEvent:
 				counter := b.Counters.Get(
-					metricName+"_counter",
+					b.suffix(metricName, "counter"),
```

Comment:
@SuperQ The whole idea of introducing the `-statsd.add-suffix` flag (instead of just always omitting automatic suffixes) was to not have a breaking change. Changing the suffix to `_total` here would be breaking though.